README.md

```diff
@@ -248,7 +250,14 @@ the input files' extensions. To format standard input, you must specify
 the `--language` and, optionally, `--query` arguments, omitting any
 input files.
 
-Note: `fmt` is a recognised alias of the `format` subcommand.
+> [!TIP]
+> `fmt` is a recognised alias of the `format` subcommand.
+
+> [!WARNING]
+> Topiary performs in-place file formatting by writing the formatted
+> output to a temporary file and then replacing that with the original
+> input file. For files with multiple links, on filesystems where that
+> makes sense, this will break the link.
```

Comment:
We _could_ test for this with `std::fs::Metadata::{nlink,number_of_links}`, but stat'ing the filesystem can be pretty expensive.

Reply:
Hmm... On reflection, `is_symlink` and `is_dir` will necessarily be stat'ing the filesystem and I'm doing this pretty indiscriminately. Let me shore this up and add the `nlink` check :+1: